jib-core/src/test/java/com/google/cloud/tools/jib/registry/ManifestPullerTest.java

```diff
@@ -89,6 +90,24 @@ public void testHandleResponse_v22()
     Assert.assertThat(manifestTemplate, CoreMatchers.instanceOf(V22ManifestTemplate.class));
   }
 
+  @Test
+  public void testHandleResponse_v22ManifestList()
+      throws URISyntaxException, IOException, UnknownManifestFormatException {
+    Path v22ManifestListFile =
+        Paths.get(Resources.getResource("core/json/v22manifest_list.json").toURI());
+    InputStream v22ManifestList = new ByteArrayInputStream(Files.readAllBytes(v22ManifestListFile));
+
+    Mockito.when(mockResponse.getBody()).thenReturn(v22ManifestList);
+    ManifestTemplate manifestTemplate =
+        new ManifestPuller<>(
+                fakeRegistryEndpointRequestProperties,
+                "test-image-tag",
+                V22ManifestListTemplate.class)
```

Comment:
I think it's good to also test when this is `ManifestTemplate.class`, which goes into a bit more interesting branch to autodetect the manifest template type.

Reply:
Makes sense.

Reply:
Still pending?

Reply:
done:https://github.com/GoogleContainerTools/jib/pull/1811/files#diff-fabb196f84239d7e62118d3d9e66aa89R112